packages/plugin-core/src/commands/ConvertLink.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from "@dendronhq/engine-server";
 import _ from "lodash";
 import { LookupControllerV3CreateOpts } from "../components/lookup/LookupControllerV3Interface";
-import { NoteLookupProviderUtils } from "../components/lookup/utils";
+import { NoteLookupProviderUtils } from "../components/lookup/NoteLookupProviderUtils";
```

Comment:
as a rule of thumb, when we have a lot of utils in a folder, can we split it out into an utils folder?

```
/lookup/utils/index.ts
/lookup/utils/NoteLookupProvider.ts
...
```

don't need to change for this one but for future PRs, lets stick to amoeba in our code base as well